project-generator/src/main/resources/io/syndesis/project/converter/templates/pom.xml.mustache

```diff
@@ -16,6 +16,7 @@
     <camel.version>@camel.version@</camel.version>
     <funktion.version>@funktion.version@</funktion.version>
     <atlasmap.version>@atlasmap.version@</atlasmap.version>
+    <syndesis-connectors.version>@syndesis-connectors.version@</syndesis-connectors.version>
```

Comment:
Remember that connectors could be at different versions, hence the need to specify the version in the dependency in the POM. I don't think taking this into a single property is safe to do here. If you do want to use properties, then each connector dependency should have it's own separate property for version.